website/source/guides/web-ui/operating-a-job.html.md

```diff
@@ -0,0 +1,218 @@
+
+The end result of this automatic procedure is a failed allocation and that failed allocation's
+rescheduled successor. Since Nomad handles all of this automatically, the Web UI makes sure to
+explain the state of allocations through iconography and linking previous and next allocations in a
+reschedule chain.
+
+[![Allocation Reschedule Icon][img-alloc-reschedule-icon]][img-alloc-reschedule-icon]
+
+[![Allocation Reschedule Details][img-alloc-reschedule-details]][img-alloc-reschedule-details]
+
+### Unhealthy Driver
+
+Given the nature of long-lived processes, it's possible for the state of the client node an
+allocation is scheduled on to change during the lifespan of the allocation. Nomad attempts to
+monitor pertinent conditions including driver health.
+
+The Web UI denotes when a driver an allocation depends on is unhealthy on the client node the
+allocation is running on.
+
+[![Allocation Unhealthy Driver][img-alloc-unhealthy-driver]][img-alloc-unhealthy-driver]
+
+### Preempted Allocations
+
+Much like how Nomad will automatically reschedule allocations, Nomad will automatically preempt
+allocations when necessary. When monitoring allocations in Nomad, it's useful to know what
+allocations were preempted and what job caused the preemption.
+
+The Web UI makes sure to tell this full story by showing which allocation caused an allocation to be
+preempted as well as the opposite: what allocations an allocation preempted. This makes it possible
+to traverse down from a job to a preempted allocation, to the allocation that caused the preemption,
+to the job that the preempting allocation is for.
+
+[![Allocation Preempter][img-alloc-preempter]][img-alloc-preempter]
+
+[![Allocation Preempted][img-alloc-preempted]][img-alloc-preempted]
+
+## Reviewing Logs for a Task
+
+A task will typically emit log information to `stdout` and `stderr`. Nomad captures these logs and
+exposes them through an API. The Web UI uses these APIs to offer `head`, `tail`, and streaming logs
+from the browser.
+
+The Web UI will first attempt to directly connecto to the client node the task is running on.
+Typically, client nodes are not accessible from the public internet. If this is the case, the Web UI
+will fall back and proxy to the client node from the server node with no loss of functionality.
+
+[![Task Logs][img-task-logs]][img-task-logs]
+
+~> Not all browsers support streaming http requests. In the event that streaming is not supported,
+logs will still be followed using interval polling.
+
+## Restarting or Stopping an Allocation or Task
+
+Ideally software always runs smoothly and as intended, but this isn't something we can count on.
+Sometimes tasks will have a memory leak, sometimes a node will have noisy neighbors, and sometimes
+we have no clue what's going so it's time to try turning it off and on again.
+
+For these times, Nomad allows for restarting and stopping individual allocations and tasks. When a
```

Comment:
I would take this first paragraph out and just start with **Nomad allows for restarting and stopping individual allocations and task**...

Reply:
Can do 👍 

Curious what your thoughts are on context setting though and where the appropriate place for that type of education is.

I wrote the first paragraph to try to get ahead of the "why would I ever want to do that?" question.

Reply:
Good question. After reading this again, I realized I actually appreciate the context you put in there. But putting the hypothetical problem scenario before stating the functionality seemed to take away from the flow when I was reading. What if we state the functionality first and then follow up with the why?

Example: 

_Nomad allows for restarting and stopping individual allocations and tasks. This is helpful in situations where there is a problem with an allocation that you need to manually resolve due to...etc._

I'm in favor of keeping your original text as well (this was just my take on it as a I read through the guide - which btw is excellent).